src/models/review.model.ts

```diff
@@ -5,7 +5,7 @@ interface IReview extends Document {
   Rating: number;
   Likes: number;
   Dislikes: number;
-  PictureURL: string;
+  PictureURL: string[];
   FacilityID: mongoose.Schema.Types.ObjectId;
```

Comment:
I think either FacilityID or UserId needs should change so they match -- i.e. UserID or FacilityId?

Reply:
what do you mean